pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  version = "feb17d";
```

Comment:
Are you sure the version doesn't break version sorting? 

I would like to see a version like 20170217_d (for the exact format, grep the repository to see other examples with dates in the version), if this assumption is invalid.